src/scss/modifiers/_state.scss

```diff
@@ -2,7 +2,7 @@
 /// @output styles for state animation for the box-styled radio inputs
 @mixin _oFormsState() {
 	.o-forms-input__state {
-		@include oTypographySize($scale: -1, $line-height: 1);
+		@include oTypographySize($scale: -1, $line-height: 1.75);
```

Comment:
How come we use a custom line height?

Reply:
Because there is a weirdness between the loading spinner having no padding and the tick icon having a padding and this was the simplest approach I found to aligning them closely 💩

Reply:
Cool, think this would benefit from a comment 👍